app/helpers/metrics_helper.rb

```diff
@@ -12,51 +12,73 @@ def sum_rollups(rollups)
     rollups.sum(&:count)
   end
 
+  def podcast_main_card_options
+    %i[downloads episodes uniques dropdays]
+  end
+
+  def episode_main_card_options
+    %i[downloads]
+  end
+
   def interval_options
     Rollups::HourlyDownload::INTERVALS.map { |i| [I18n.t(".helpers.label.metrics.interval.#{i.downcase}"), i] }
   end
 
-  def date_range_options(episode = nil)
+  def date_preset_options(episode = nil)
     if episode
-      episode_date_presets(episode)
+      episode_date_preset_options
     else
-      podcast_date_presets
+      podcast_date_preset_options
     end
   end
 
-  def podcast_date_presets
-    metrics_date_presets(Rollups::HourlyDownload::PODCAST_DATE_PRESETS)
+  def podcast_date_preset_options
+    Rollups::HourlyDownload::PODCAST_DATE_PRESETS.map do |preset|
+      [label_for_date_preset(preset), preset.to_s]
+    end
   end
 
-  def episode_date_presets(episode)
-    metrics_date_presets(Rollups::HourlyDownload::EPISODE_DATE_PRESETS, episode)
+  def episode_date_preset_options
+    Rollups::HourlyDownload::EPISODE_DATE_PRESETS.map do |preset|
+      [label_for_date_preset(preset), preset.to_s]
+    end
   end
 
-  def metrics_date_presets(options, episode = nil)
-    options.map do |opt|
-      count, interval, type = opt.to_s.split("_")
+  def dates_from_preset(preset, episode = nil)
```

Comment:
given a preset, this method now just returns a `date_start` and `date_end`. there is a little bit of difference between Podcast/Episode Metrics of the default `start_date` return value, but otherwise these are calculated with the helper methods below.